channel.go

```diff
@@ -370,7 +370,10 @@ func (ch *Channel) serve() {
 				if max := 1 * time.Second; acceptBackoff > max {
 					acceptBackoff = max
 				}
-				ch.log.Warnf("accept error: %v; retrying in %v", err, acceptBackoff)
+				ch.log.WithFields(
+					LogField{"error", err.Error()},
+					LogField{"backoff", acceptBackoff},
+				).Warn("Accept error, will wait and retry.")
```

Comment:
Was intentional on my part - I think of logs as letters to future me. All Warn, Error, and Fatal logs in tchannel-go should now have terminal punctuation.

I haven't noticed a strong convention in go-common re: initial capitals and sentence-style punctuation - does Google recommend a particular logging style? I'm happy to change if necessary.